vips/vip-499/bscmainnet.ts

```diff
@@ -0,0 +1,388 @@
+      },
+      {
+        target: ACM_ARBITRUM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "setSnapshot(uint256,uint256)", NORMAL_TIMELOCK_ARBITRUM],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ACM_ARBITRUM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "setGrowthRate(uint256,uint256)", NORMAL_TIMELOCK_ARBITRUM],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ACM_ARBITRUM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "setSnapshotGap(uint256)", NORMAL_TIMELOCK_ARBITRUM],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ACM_ARBITRUM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "setSnapshot(uint256,uint256)", CRITICAL_TIMELOCK_ARBITRUM],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ACM_ARBITRUM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "setGrowthRate(uint256,uint256)", CRITICAL_TIMELOCK_ARBITRUM],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ACM_ARBITRUM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "setSnapshotGap(uint256)", CRITICAL_TIMELOCK_ARBITRUM],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ACM_ARBITRUM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "setSnapshot(uint256,uint256)", FASTTRACK_TIMELOCK_ARBITRUM],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ACM_ARBITRUM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "setGrowthRate(uint256,uint256)", FASTTRACK_TIMELOCK_ARBITRUM],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ACM_ARBITRUM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "setSnapshotGap(uint256)", FASTTRACK_TIMELOCK_ARBITRUM],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: weETH_ORACLE_ARBITRUM,
+        signature: "setSnapshot(uint256,uint256)",
+        params: [increaseExchangeRateByPercentage(weETH_Initial_Exchange_Rate, BigNumber.from("44")), 1747682525],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: weETH_ORACLE_ARBITRUM,
+        signature: "setGrowthRate(uint256,uint256)",
+        params: [parseUnits("0.053", 18), DAYS_30],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: weETH_ORACLE_ARBITRUM,
+        signature: "setSnapshotGap(uint256)",
```

Comment:
Snapshot gap should be defined in terms of the exchange rate, not as a percentage. So, in this case, I think the right value would be:

```
weETH_Initial_Exchange_Rate.mul(44).div(10000)
```

Reply:
We should apply the same change in the rest of the commands `setSnapshotGap`